### ISSUE TITLE
Floor boundary showing up in Holographic Emulation mode

### ISSUE THREAD

**Issue by genereddick** [microsoft/MixedRealityToolkit-Unity#3803]
## Overview
Started getting this odd behavior after updating this morning. Testing on WMR headset the floor boundary started working (wasn't showing up for me prior to today). So can teleport around the floor plain etc.

However, turning off the WMR portal and testing using Holographic Emulation (Simulate in Editor, which I believe is meant to simulate a transparent headset -- at least it is using the transparent camera settings) and the floor boundary display is showing up in the editor window at head height.

## Expected Behavior
No floor boundary

## Actual Behavior
Floor boundary shows up

## Steps to reproduce
_(Links to sample github project preferred)_

## Unity Editor Version
2018.3.11f1

## Mixed Reality Toolkit Release Version
mrtk_development as of earlier this morning

**Comment by davidkline-ms** [comment]
Thanks for the report! We will investigate as soon as possible.

**Comment by genereddick** [comment]
After a reboot, I started the same project in Holographic Emulation | Simulate in Editor and the Floor Boundary was not there, so working as expected.

I started the WM Portal, ran the project, Floor Boundary showed up as expected (on headset).

Stopped, re-ran in Holographic Emulation  | Simulate in Editor, Floor Boundary is there at head height.

Stopped, shut down the WM Portal, re-ran in Holographic Emulation | Simulate in Editor, Floor Boundary is there at head height.

So, the pattern seems to be once I run in the WM Portal whatever code decides that the Floor Boundary needs to be shown isn't detecting the change in output.
